.travis.yml

```diff
@@ -6,12 +6,18 @@ cache:
     - $HOME/.ivy2/cache
     - $HOME/.sbt/boot/
 
-scala:
-- 2.11.7
-
 jdk:
```

Comment:
Can we build against openjdk8?

Reply:
Yeah, in line 10 you have defined openjdk8 for all versions

Reply:
Changed